www/index.html

```diff
@@ -6,6 +6,7 @@
     <link href="//netdna.bootstrapcdn.com/font-awesome/4.0.3/css/font-awesome.min.css" rel="stylesheet" />
     <link href='http://fonts.googleapis.com/css?family=Condiment|Roboto:100,300' rel='stylesheet' type='text/css' />
     <link href="main.css" rel="stylesheet" />
+    <link href='http://fonts.googleapis.com/css?family=Pacifico' rel='stylesheet' type='text/css'>
```

Comment:
You can just add the Pacifico font family to the import on [line 7](https://github.com/openmcac/landing/commit/122a1750c1823fdaabd9118599c9fa52d9c335a4#diff-ab469c2203f00940178ab4d78eacaad6R7) above.